src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import BookForm from './components/BookForm';
+import Categories from './components/Categories';
+import Navigation from './components/Navigation';
+
+function App() {
+  return (
+    <>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<BookForm />} />
```

Comment:
- [ ] as part of the requirements the root path should display the book component and not BookForm kindly update it based on the detailed comment below 🙏